src/python/grpcio/grpc/_channel.py

```diff
@@ -374,13 +387,24 @@ def details(self):
                 self._state.condition.wait()
             return _common.decode(self._state.details)
 
+    def debug_error_string(self):
+        with self._state.condition:
+            while self._state.debug_error_string is None:
+                self._state.condition.wait()
+            return _common.decode(self._state.debug_error_string)
+
     def _repr(self):
         with self._state.condition:
             if self._state.code is None:
                 return '<_Rendezvous object of in-flight RPC>'
             else:
-                return '<_Rendezvous of RPC that terminated with ({}, {})>'.format(
-                    self._state.code, _common.decode(self._state.details))
+                if self._state.code == grpc.StatusCode.OK:
```

Comment:
Use `is` and `is not` when checking whether or not a field is assigned to a singleton object.